tiledb/sm/storage_manager/storage_manager.cc

```diff
@@ -1943,7 +1944,16 @@ Status StorageManager::read(
   RETURN_NOT_OK(vfs_->read(uri, offset, buffer->data(), nbytes));
   buffer->set_size(nbytes);
   buffer->reset_offset();
+  return Status::Ok();
+}
 
+Status StorageManager::read(
```

Comment:
We shouldn't add this API. Tile::alloc_data needs to go away eventually. Tiles will always be created with the right size from the get go to achieve C.41 compliance.

Reply:
avoided